test/runtests.jl

```diff
@@ -11,6 +11,8 @@ else
     using Test
 end
 
+!isdefined(Base, :Base64) && using Base64
```

Comment:
There's already a `VERSION >= v"0.7.0" && using Base64` above.   But maybe this can be removed?   I don't see any usage of the `Base64` module in the test script.

The whole `using Test` logic can also be replaced with `using Compat.Test`

Reply:
OK, sorry I missed that when merging. Base64 is needed for [`stringmime`](https://github.com/JuliaPy/PyPlot.jl/blob/e46a33d327a11985e32db6c966513a82752b5266/test/runtests.jl#L24).